src/main/java/pl/osowicz/task_manager/Task/TaskController.java

```diff
@@ -27,16 +29,16 @@ public String home() {
         return "index";
     }
 
-    @GetMapping("/addTask")
+    @GetMapping("/task/add")
     public String addTask(Model model) {
-        List<User> users = userRepository.findAll();
+        List<User> users = userRepository.findAllByActiveIsTrue();
```

Comment:
findAllByActive(true) - zawsze lepiej dawać wartość jako argument niż tworzyć metodę IsTrue - bo może w innym miejscu będziesz chciał nieaktywnych pobrać i już będziesz miał to gotowe, bez duplikacji